### ISSUE TITLE
Accessibility

### ISSUE THREAD

**Issue by rachael-t** [rachael-t/riverside-resort#15]
- [x] Accessibility audits should be at 100%
- [x] A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- [x] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader
- [x] Application should be fully responsive for mobile - tablet - desktop

**Comment by rachael-t** [comment]
All functionality works across Chrome, Firexfox, and Safari.
* Tabbing does not currently work in Safari.

**Comment by rachael-t** [comment]
<img width="1295" alt="Screen Shot 2020-04-21 at 3 26 35 PM" src="https://user-images.githubusercontent.com/54180641/79915647-97283c00-83e4-11ea-8eef-83b3ede7f3ae.png">

**Comment by rachael-t** [comment]
Re: Safari tabbing issue -> discussed with instructor "fair to assume users that are keyboard-only know about this Safari issue and have adjusted their computer preferences to account for it"

**Comment by rachael-t** [comment]
Got responsive layout to work, but cannot import mediaqueries file - this is how it works under base.scss:

<img width="474" alt="Screen Shot 2020-04-21 at 4 20 28 PM" src="https://user-images.githubusercontent.com/54180641/79923588-0efd6300-83f3-11ea-853b-a627f9563fb3.png">
